lib/resolver.coffee

```diff
@@ -41,6 +42,9 @@ yeomanResolver = require('yeoman-environment/lib/resolver')
 exports.getNodeModulesPaths = ->
 	paths = yeomanResolver.getNpmPaths()
 
+	if os.platform() isnt 'win32'
+		paths.unshift('/usr/local/lib/node_modules')
```

Comment:
It would probably make more sense to use `require('module').globalPaths` or something similar to get a list of all the global paths rather than hardcoding them